src/lib.rs

```diff
@@ -64,8 +64,28 @@
 
 #![cfg_attr(not(feature = "std"), no_std)]
 
-macro_rules! define_vector {
-    ($name:ident, $mint_type:ident, $align:literal, $ty:ty, $count:literal<- $doc:literal) => {
+macro_rules! define_vectors {
+    ( $(( $name:ident, $mint_name:ident, $prim:ident * $count:literal, align: $align:literal, size: $size:literal ),)* ) => {
+        $(
+            define_vectors!(@impl
+                $name,
+                mint::$mint_name<$prim>,
+                $align,
+                $prim,
+                $count,
+                concat!(
+                    "Vector of ", stringify!($count), " `", stringify!($prim), "`. ",
```

Comment:
nit: should be 
```
" `", stringify!($prim), "`s. "
```

Reply:
Hm. Pluralizing a type like this comes off as strange to me. Is there an example of this style somewhere in Rust docs, like in `std`?

Reply:
It can go either way, I just find it more natural to pluralize it. Doesn't really matter, feel free to ignore.

Reply:
Maybe the better way to pluralize would be something like "Vector of `f32` values`. How does that sound?

Reply:
Sounds good